Source/CardView.h

```diff
@@ -25,5 +25,7 @@
 @optional
```

Comment:
Yeah I saw it, just didn't saw why they where useful, I didn't knew that in your app you had the requirement of removing and adding cards on the flight. It feels very stiff, I always thought this could be more direct manipulation instead of buttons that you had to press.

Take a look at this: https://github.com/jonahgrant/wwdc